tools/c7n_kube/pyproject.toml

```diff
@@ -26,3 +26,6 @@ vcrpy = "^4.0.2"
 [build-system]
 requires = ["poetry>=0.12", "setuptools"]
 build-backend = "poetry.masonry.api"
+
+[tool.poetry.scripts]
+"c7n-adm" = "c7n_kube.cli:cli"
```

Comment:
definitely want a rename on this, adm is super ambitious to provider.. how about c7n-kates ;-)